pkg/asset/internal/templates.go

```diff
@@ -430,6 +437,40 @@ spec:
     type: RollingUpdate
 `)
 
+var CheckpointerServiceAccount = []byte(`apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: kube-system
+  name: pod-checkpointer
+`)
+
+var CheckpointerRole = []byte(`apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Note: if the pod-checkpointer only watches kube-system (e.g. its own namespace) we can drop this to a Role.

Reply:
I think this is ok to keep for now but add to the checkpointer README and the release note.